data-prepper-plugins/service-map-stateful/src/main/java/org/opensearch/dataprepper/plugins/processor/ServiceMapProcessorConfig.java

```diff
@@ -5,8 +5,20 @@
 
 package org.opensearch.dataprepper.plugins.processor;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+
 public class ServiceMapProcessorConfig {
     static final String WINDOW_DURATION = "window_duration";
```

Comment:
Please make this `private` now. It doesn't need to be used outside of this class with this change.